### DISCUSSION TITLE
Filter by entry point

### DISCUSSION THREAD

**Discussion by chriskrogh** [webpack-contrib/webpack-bundle-analyzer#622] [Ideas]
At Faire, we use [@next/bundle-analyzer](https://www.npmjs.com/package/@next/bundle-analyzer) (which is just a [wrapper around webpack-bundle-analyzer](https://github.com/vercel/next.js/blob/76def31fb208d2df70a707edbc7a9ab9945d58cc/packages/next-bundle-analyzer/index.js#L7C12-L7C12)) to understand our page weight. 
We currently have > 450 "routes" in our next app and the current entrypoint dropdown makes it really difficult for teams to find the route they care about.

it'd be great if we could filter the list of entry points by typing into the input before selecting an option.

**Comment by valscion** [comment]
Hi! I'm not sure I follow — are you able to post some screenshots or some videos of how the analyzer looks like to you and how it's difficult with this big amount of "routes"?

**Reply by chriskrogh** [reply]
sure. here's a screen recording. this is on a 16" MBP. 

https://github.com/webpack-contrib/webpack-bundle-analyzer/assets/35310373/8bebb619-f6e3-4578-84e2-2d426fb3d010


the ask here is to be able to type into the input to filter the list (instead of using a simple [select tag](https://github.com/webpack-contrib/webpack-bundle-analyzer/blob/f01056a51fa16f3274204b5b98bba1be3a3f496d/client/components/Dropdown.jsx#L15))

because the list is so long, I find myself needing to inspect element, copy all of the option elements into a text file where I can search for the route I care about. then bisect the list in the UI to eventually find my route.

**Reply by valscion** [reply]
Oh yeah thanks, now I see!

This select dropdown was added in here:

* https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/519

@pas-trop-de-zele would you be interested in trying to make this select dropdown more accessible when the contents of that dropdown get as large as @chriskrogh has here?

@chriskrogh you can also take a look at figuring this out on your own if you'd so want to.

**Reply by chriskrogh** [reply]
I can give it a go this weekend if @pas-trop-de-zele doesn't get to it before then

**Reply by chriskrogh** [reply]
https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/624

**Comment by valscion** [comment]
The pull request implementing this idea was released in v4.10.0! Thanks again, @chriskrogh ☺️ 

* #624
